src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/swift/SwiftPermanentStorage.java

```diff
@@ -54,8 +68,9 @@ public SwiftPermanentStorage(Properties properties) throws PermanentStorageExcep
 
     private boolean checkProperties(Properties properties) {
         String[] propertiesSet = {
-                SapsPropertiesConstants.PERMANENT_STORAGE_TASKS_DIR,
-                SapsPropertiesConstants.SWIFT_CONTAINER_NAME
+            SapsPropertiesConstants.PERMANENT_STORAGE_TASKS_DIR,
```

Comment:
I don't like this use of properties so deep into the code. Didn't we agree on parsing properties only in the constructor? (and, using varilables after that)?

Reply:
Yes. I removed properties field from there.